tests/versioned/transaction-naming-tests.js

```diff
@@ -136,6 +136,30 @@ function createTransactionTests(t, frameworkName) {
     })
   })
 
+  t.test(
+    'anonymous query, only returns reserved field(id) should return deepest unique path',
+    (t) => {
+      const { helper, serverUrl } = t.context
+
+      const query = `query {
+      searchCollection(title: "True life") {
+        id
+      }
+    }`
```

Comment:
looking at this indentation i'm surprised this is passing linting. i wonder if this is mixing spaces and tabs and if our new base linting rule isn't enforcing spaces? or maybe the GitHub renderer is rendering incorrect?

Reply:
looks like the back-tick style string just doesn't get flagged for the indentation. its all spaces. prob cause the rules cause some issues in other cases. anyways, not super important just stood out in a simple diff.

Reply:
hmm. i can look into a linting rule. i'd think prettier would fix this